lookup.js

```diff
@@ -0,0 +1,182 @@
+  { name: "Ninetales" },
+  { name: "Jigglypuff" },
+  { name: "Wigglytuff" },
+  { name: "Zubat" },
+  { name: "Golbat" },
+  { name: "Oddish" },
+  { name: "Gloom" },
+  { name: "Vileplume" },
+  { name: "Paras" },
+  { name: "Parasect" },
+  { name: "Venonat" },
+  { name: "Venomoth" },
+  { name: "Diglett" },
+  { name: "Dugtrio" },
+  { name: "Meowth" },
+  { name: "Persian" },
+  { name: "Psyduck" },
+  { name: "Golduck" },
+  { name: "Mankey" },
+  { name: "Primeape" },
+  { name: "Growlithe" },
+  { name: "Arcanine" },
+  { name: "Poliwag" },
+  { name: "Poliwhirl" },
+  { name: "Poliwrath" },
+  { name: "Abra" },
+  { name: "Kadabra" },
+  { name: "Alakazam" },
+  { name: "Machop" },
+  { name: "Machoke" },
+  { name: "Machamp" },
+  { name: "Bellsprout" },
+  { name: "Weepinbell" },
+  { name: "Victreebel" },
+  { name: "Tentacool" },
+  { name: "Tentacruel" },
+  { name: "Geodude" },
+  { name: "Graveler" },
+  { name: "Golem" },
+  { name: "Ponyta" },
+  { name: "Rapidash" },
+  { name: "Slowpoke" },
+  { name: "Slowbro" },
+  { name: "Magnemite" },
+  { name: "Magneton" },
+  { name: "Farfetch'd" },
+  { name: "Doduo" },
+  { name: "Dodrio" },
+  { name: "Seel" },
+  { name: "Dewgong" },
+  { name: "Grimer" },
+  { name: "Muk" },
+  { name: "Shellder" },
+  { name: "Cloyster" },
+  { name: "Gastly" },
+  { name: "Haunter" },
+  { name: "Gengar" },
+  { name: "Onix" },
+  { name: "Drowzee" },
+  { name: "Hypno" },
+  { name: "Krabby" },
+  { name: "Kingler" },
+  { name: "Voltorb" },
+  { name: "Electrode" },
+  { name: "Exeggcute" },
+  { name: "Exeggutor" },
+  { name: "Cubone" },
+  { name: "Marowak" },
+  { name: "Hitmonlee" },
+  { name: "Hitmonchan" },
+  { name: "Lickitung" },
+  { name: "Koffing" },
+  { name: "Weezing" },
+  { name: "Rhyhorn" },
+  { name: "Rhydon" },
+  { name: "Chansey" },
+  { name: "Tangela" },
+  { name: "Kangaskhan" },
+  { name: "Horsea" },
+  { name: "Seadra" },
+  { name: "Goldeen" },
+  { name: "Seaking" },
+  { name: "Staryu" },
+  { name: "Starmie" },
+  { name: "Mr Mime" },
+  { name: "Scyther" },
+  { name: "Jynx" },
+  { name: "Electabuzz" },
+  { name: "Magmar" },
+  { name: "Pinsir" },
+  { name: "Tauros" },
+  { name: "Magikarp" },
+  { name: "Gyarados" },
+  { name: "Lapras" },
+  { name: "Ditto" },
+  { name: "Eevee" },
+  { name: "Vaporeon" },
+  { name: "Jolteon" },
+  { name: "Flareon" },
+  { name: "Porygon" },
+  { name: "Omanyte" },
+  { name: "Omastar" },
+  { name: "Kabuto" },
+  { name: "Kabutops" },
+  { name: "Aerodactyl" },
+  { name: "Snorlax" },
+  { name: "Articuno" },
+  { name: "Zapdos" },
+  { name: "Moltres" },
+  { name: "Dratini" },
+  { name: "Dragonair" },
+  { name: "Dragonite" },
+  { name: "Mewtwo" },
+  { name: "Mew" },
+];
+
+/**
+ * finds a valid pokedex number for the name given, or validates range for pokedex number
+ * returns falsey/null when input invalid
+ * @param {String|Number} pokemonInput
+ */
+function findPokedexNumber(pokemonInput) {
```

Comment:
Because https://pokemon.alexonsager.net gives us missingno I think we can remove the validation from here.

Something like this...

`function findPokedexNumber(pokemon = "") {
  const index = pokedex.findIndex(
    (x) => x.name.toLowerCase() === pokemon.toLowerCase()
  );

  if (index >= 0) return index + 1;

  return pokemon;
}`

Reply:
won't this remove the original behaviour, which was to specify number? Or did you want to keep that?

Reply:
Shouldn't remove it. If it can't find an index it returns the original input.

Reply:
Ah, I see now. Do you think there's value in forcing a missingno when the user specifies a bad name e.g. 
```
findPokedexNumber("notarealpokemon"); 
```
Or do you think it's better to just pass that directly to the api ?

Reply:
I think it's better to forward it to the API